src/Microsoft.AspNet.Authentication.OAuth/OAuthAuthenticationOptions.cs

```diff
@@ -77,7 +69,7 @@ public string Caption
         /// <value>
         /// The back channel timeout.
         /// </value>
-        public TimeSpan BackchannelTimeout { get; set; }
+        public TimeSpan BackchannelTimeout { get; set; } = TimeSpan.FromSeconds(60);
```

Comment:
Unrelated change, but I'd like to make all these options virtual, so inheritors can replace the documentation and set values inline:

`/// <summary>My different summary</summary>
public override TimeSpan BackchannelTimeout { get; set; } = TimeSpan.FromSeconds(120);`